resdk/query.py

```diff
@@ -239,6 +239,10 @@ def _fetch(self):
             raise NotImplementedError(
                 'Unsupported query_method: {}'.format(self.resource.query_method))
 
+        # Extract data from paginated response
+        if len(items) == 4 and set(items) == set(['next', 'previous', 'count', 'results']):
```

Comment:
How could it contain a `results` key by accident? I think this is something that is defined by the API and SDK should expect the server to return the correct response?

Also, if `items` contains a `results` key by accident, then `items` is still a dictionary and everything from there forward will be incorrect as it expects a list.